.drone.yml

```diff
@@ -198,77 +195,58 @@ trigger:
       - refs/tags/**
 
 steps:
-  - name: init
-    image: quay.io/sighup/e2e-testing-drone-plugin:v1.26.4
+  - name: create-kind-cluster
+    image: quay.io/sighup/dind-kind-kubectl-kustomize:0.20.0_1.29.1_3.10.0
     pull: always
     volumes:
+      - name: dockersock
+        path: /var/run/docker.sock
       - name: shared
         path: /shared
```

Comment:
you don't need the shared volume. The `workdir` (`/drone/src`) where the repo is cloned is shared between the pipelines
```suggestion
```